assets/js/modules/analytics/components/module/ModuleAcquisitionChannelsWidget/PieChart.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * PieChart component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import GoogleChart from '../../../../../components/GoogleChart';
+import { extractAnalyticsDataForPieChart } from '../../../util';
+import PropTypes from 'prop-types';
```

Comment:
Please, move it to external dependendencies.

Reply:
Updated in https://github.com/google/site-kit-wp/pull/3035/commits/a708e6d6428d8b7133e4fe2d7ef40a8cc0df262d